chatlas/_chat.py

```diff
@@ -23,6 +24,11 @@
     overload,
 )
 
+from mcp import (
+    ClientSession as MCPClientSession,
+)
+from mcp.client.sse import sse_client
+from mcp.client.stdio import StdioServerParameters, stdio_client
```

Comment:
I think these would have to be run time imports given `mcp` is an optional dependency